governance/polls/SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - May 27, 2024.md

```diff
@@ -0,0 +1,61 @@
+---
+title: SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - May 27, 2024
+summary: Signal your support or opposition to increasing the DAI supply caps for Morpho Spark DAI Vault on Ethereum mainnet.
+discussion_link: https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-05-27T16:00:00
+end_date: 2024-05-30T16:00:00
+---
+# Poll: SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - May 27, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Scope Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, May 27 at 16:00 UTC.
```

Comment:
```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, May 27 at 16:00 UTC.
```
I'm removing "Scope" as it is only used in the term Stability Scope Facilitator and not in Governance Facilitators, to keep the document consistent